tests/unit_tests/test_gameconnection.py

```diff
@@ -415,7 +415,8 @@ async def test_handle_action_GameOption(
     assert game.max_players == 7
     # I don't know what these paths actually look like
     await game_connection.handle_action("GameOption", ["ScenarioFile", "C:\\Maps\\Some_Map"])
-    assert game.map_file_path == "maps/some_map.zip"
+    assert game.map_name == "some_map"
+    assert game.map_scenario_path == "C:/Maps/Some_Map"
```

Comment:
since game is a mock the setter is not invoked and map_file_path is not set by the setter